discovery-provider/alembic/versions/ac0c6f94dd8d_partial_created_at_idx.py

```diff
@@ -0,0 +1,85 @@
+"""partial_created_at_idx
+
+Revision ID: ac0c6f94dd8d
+Revises: df4709b4391b
+Create Date: 2022-03-03 21:11:42.415590
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ac0c6f94dd8d"
+down_revision = "df4709b4391b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    query = """
+
+    -- make track_owner_id_idx a partial index
+    DROP INDEX track_owner_id_idx;
+    CREATE INDEX track_owner_id_idx ON tracks (owner_id) WHERE is_current and not is_delete;
+
+    -- add index for track_created_at
+    CREATE INDEX track_created_at_idx ON tracks (created_at) WHERE is_current and not is_delete;
+
+    -- add index for playlist owner
+    CREATE INDEX playlist_owner_id_idx ON playlists (playlist_owner_id) WHERE is_current and not is_delete;
+
+    -- add index for playlist_created_at
+    CREATE INDEX playlist_created_at_idx ON playlists (created_at) WHERE is_current and not is_delete;
+
+    -- make repost_user_id a partial index
+    DROP INDEX repost_user_id_idx;
+    CREATE INDEX repost_user_id_idx ON reposts (user_id, repost_type) WHERE is_current and not is_delete;
+
+    -- make repost_item_id a partial index
+    DROP INDEX repost_item_id_idx;
+    CREATE INDEX repost_item_id_idx ON reposts (repost_item_id, repost_type) WHERE is_current and not is_delete;
+
+    -- add index for repost_created_at
+    CREATE INDEX repost_created_at_idx ON reposts (created_at) WHERE is_current and not is_delete;
+
+    -- make followee_user_id a partial index
+    DROP INDEX ix_follows_followee_user_id;
+    CREATE INDEX ix_follows_followee_user_id ON follows (followee_user_id) WHERE is_current and not is_delete;
```

Comment:
Is there a difference between creating an index on followee_user_id and follower_user_id separately vs together? I'm guessing most of them time they'd be queried together.

Reply:
There's actually a fair number of places where we query just one side... so I think it makes sense to keep as two separate indexes.